src/helper.js

```diff
@@ -1,17 +1,26 @@
 var path = require("path");
 var os = require("os");
 
+var HOME = expandHome("~/.lounge");
+
 module.exports = {
-	HOME: (process.env.HOME || process.env.USERPROFILE) + "/.lounge",
-	getConfig: getConfig,
+	HOME: HOME,
+	CONFIG_PATH: path.join(HOME, "config.js"),
```

Comment:
We discussed this in IRC, but for sake of visibility: this breaks `--home` option.